broker_app.py

```diff
@@ -26,6 +26,7 @@
 logging.getLogger('ingest').setLevel(logging.INFO)
 logging.getLogger('ingest.api.ingestapi').setLevel(logging.INFO)
 logging.getLogger('broker.service.spreadsheet_upload_service').setLevel(logging.INFO)
+logging.getLogger('broker.submissions.export_to_spreadsheet_service').setLevel(logging.INFO)
 logging.getLogger('geo_to_hca').setLevel(logging.INFO)
```

Comment:
Is there a reason we are not using a configuration file for setting logging levels?

Reply:
Added. Great suggestion!